src/main/java/net/pms/util/ProcessUtil.java

```diff
@@ -217,8 +217,13 @@ public static String run(String... cmd) {
 	// Whitewash any arguments not suitable to display in dbg messages
 	// and make one single printable string
 	public static String dbgWashCmds(String[] cmd) {
-		for(int i=0; i < cmd.length; i++) {
-			if(cmd[i].contains("headers")) {
+		for (int i=0; i < cmd.length; i++) {
+			// Wrap arguments with spaces in double quotes to make them runnable if copy-pasted
+			if (cmd[i].contains(" ")) {
```

Comment:
I think this should be tested on multiple operating systems before merging, since we made a change years ago that was similar and broke transcoding on some... I can't remember details.